### PR TITLE
Remove variants, update commit id

### PR DESCRIPTION
**Pull request by vanhauser-thc**
This PR removes all the variants and updates the commit IDs, optimized optimal further, enabled cmplog for base afl++.
It also fixes afl++ for proj4 with cmplog LTO which failed due an update in the proj4 fuzzer.

### CODE REVIEW

**Comment by vanhauser-thc** [comment]
afl - not touched CI issue

afl_qemu - not touched, CI issue

aflplusplus_qemu - failed to build LLB: mount already exists with name

aflplusplus + aflplusplus_optimal dunno both say "This file has been truncated. View the full logs in the menu once the check is completed." so it could be bug in the afl++ setup or could be a CI issue.

**Comment by vanhauser-thc** [comment]
new commit that:
 - adds the weizz fuzzer (as greybox binary only variant)
 - updates honggfuzz
 - adds three dummy fuzzer that are exactly the same, also in their PRNG to see how much variance there is single in fuzzbench runs

plus adding a new testrun.

**Comment by vanhauser-thc** [comment]
all green except weizz. weizz complains about missing (or bad?) seeds in one benchmark and crashs in another.
I would propose it is merged and run, most benchmarks should succeed and that gives a good first indication how weizz performs.

**Comment by vanhauser-thc** [comment]
please do not accept the PR in the next hour, maybe I will add something exciting until then

**Comment by vanhauser-thc** [comment]
This new commit updates the weizz fuzzer to build for a few more targets. still  of them weizz will crash on, but that is out of the scope for me to get it running :) once it has a run andrea can have a look.

New: afl++ with a qemu first level tainter, which taints new queue entries for the input usage and then the queue input is selectivly only fuzzed on the touches bytes and on bytes that were newly found compared to the previous entry.
Not sure this variant will work with all targets. if not ... doesnt matter. 
I just need a look on a few graphs if this method is actually useful or if my last 4 days were a waste of time.

No other fuzzers were touched, so if any other fails come up, this is the build CI/build system acting up.

please merge and run the experiment :)

@ jonathanmetzman 
if Jonathan is sick can you take care of this @lszekeres @inferno-chromium?

thanks!

File: fuzzers/aflplusplus_taint/builder.Dockerfile [hunk 1]
```diff
@@ -15,18 +15,22 @@
 ARG parent_image
 FROM $parent_image
 
+RUN cd / && echo Test
```
Comment (inferno-chromium) [inline fuzzers/aflplusplus_taint/builder.Dockerfile:18]: nit: Can be removed.

File: fuzzers/weizz/runner.Dockerfile [hunk 2]
```diff
@@ -13,3 +13,11 @@
 # limitations under the License.
 
 FROM gcr.io/fuzzbench/base-image
+
+# honggfuzz requires libfd and libunwid
```
Comment (inferno-chromium) [inline fuzzers/weizz/runner.Dockerfile:17]: nit: "libunwind."

File: fuzzers/weizz/fuzzer.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+def build():
+    """Build benchmark."""
+    os.environ['CC'] = 'clang'
+    os.environ['CXX'] = 'clang++'
+    os.environ['FUZZER_LIB'] = '/libQEMU.a'
+
+    utils.build_benchmark()
+
+    # Copy over honggfuzz's main fuzzing binary.
```
Comment (inferno-chromium) [inline fuzzers/weizz/fuzzer.py:31]: nit: weizz

File: fuzzers/aflplusplus_taint/runner.Dockerfile [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/fuzzbench/base-image
+
+ENV LD_LIBRARY_PATH /out
```
Comment (inferno-chromium) [inline fuzzers/aflplusplus_taint/runner.Dockerfile:17]: nit: duplicate line below.

**Comment by vanhauser-thc** [comment]
done :)

**Comment by inferno-chromium** [comment]
> done :)

you define real-time Sir!

**Pull request merged**